driver/razermouse_driver.c

```diff
@@ -1456,6 +1478,16 @@ static ssize_t razer_attr_write_poll_rate(struct device *dev, struct device_attr
         break;
     }
 
+    //Same as above except for the transaction ID
+    switch(device->usb_pid) {
+    case USB_DEVICE_ID_RAZER_VIPER_MINI_SE_WIRED:
+    case USB_DEVICE_ID_RAZER_VIPER_MINI_SE_WIRELESS:
+        request = razer_chroma_misc_set_polling_rate2(polling_rate, 0x01);
+        request.transaction_id.id = 0x1F;
+        razer_send_payload(device, &request, &response);
+        break;
+    }
```

Comment:
Move this into the same `switch` above since it's doing essentially the same (with different transaction_id as you noted) but just for a different device. So below the `break` add your `case` statements plus your code.

Reply:
Gotcha. Not entirely sure why I made that a separate switch. Sorry about that.